js/mail.js

```diff
@@ -4,6 +4,7 @@ var Mail = {
 		currentFolderId: null,
 		currentAccountId: null,
 		currentMessageId: null,
+                currentlyLoading: null,
```

Comment:
So, can you fix it? :) Then we can merge it I guess. @wurstchristoph did you test it and also works for you?